plugin.xml

```diff
@@ -5,13 +5,22 @@
     <label lang="de">Backend Bestellungen</label>
     <label lang="en">Backend orders</label>
 
-    <version>3.0.2</version>
+    <version>3.0.3</version>
     <copyright>(c) by shopware AG</copyright>
     <license>proprietary</license>
-    <link>http://store.shopware.com</link>
+    <link>https://store.shopware.com</link>
     <author>shopware AG</author>
     <compatibility minVersion="5.6.0" maxVersion="5.99.99"/>
 
+    <changelog version="3.0.3">
+        <changes lang="de">
+            PT-13027 - Hinzufügen der Variable sTaxRates zur sOrder-Mail;
```

Comment:
```suggestion
            PT-13027 - Die Steuern sind nun im Template der Bestellbestätigungsmail verfügbar;
```

please make sure, that this is not to technical, as this is for the shop owners/ merchants